packages/graphback-cli/src/helpers/generate.ts

```diff
@@ -21,61 +22,65 @@ followed by ${chalk.cyan(`${cliName}db`)} to create database.
  * Generate schema and resolvers using graphback-core and
  * write them into generated folder
  */
-export async function generateBackend(): Promise<void> {
+export async function generateBackend(pathForModel: string): Promise<void> {
   try {
-    const models = new GlobSync('model/*.graphql', { cwd: process.cwd() })
+    const configPath = `${process.cwd()}/graphback.json`
+
+    const { paths, database, generation, client } = JSON.parse(readFileSync(configPath, "utf8"))
+
+    const models = new GlobSync(`${pathForModel}/*.graphql`)
 
     if (models.found.length === 0) {
-      logError(`No graphql file found inside ${process.cwd()}/model folder.`)
+      logError(`No graphql file found inside ${pathForModel} folder.`)
       process.exit(0)
     }
-    const configPath = `${process.cwd()}/config.json`
-
-    const { database, generation, client } = JSON.parse(readFileSync(configPath, "utf8"))
 
-    const path: string = process.cwd()
-    const schemaText: string = models.found.map((m: string) => readFileSync(`${path}/${m}`, 'utf8')).join('\n')
+    const schemaText: string = models.found.map((m: string) => readFileSync(`/${m}`, 'utf8')).join('\n')
+    
+    const pathForSchema: string = join(process.cwd(), paths.schema)
+    const outputSchemaPath: string = `${pathForSchema}/generated.ts`
 
-    const outputSchemaPath: string = `${process.cwd()}/src/schema/generated.ts`
-    const outputResolverPath: string = `${process.cwd()}/src/resolvers`
+    const customResolvers: string = join(process.cwd(), paths.customResolvers)
+    const generatedResolvers: string = join(process.cwd(), paths.generatedResolvers)
 
     const backend: GraphQLBackendCreator = new GraphQLBackendCreator(schemaText, generation)
-
     const generated: IGraphQLBackend = await backend.createBackend(database)
-    
+
     let generatedClient
     let clientPath: string
-    if(client) {
+
+    if (client) {
       generatedClient = await backend.createClient()
-      clientPath = `${process.cwd()}/client`
+      clientPath = join(process.cwd(), paths.client)
       if(!existsSync(clientPath)) {
-        mkdirSync(clientPath)
+        mkdirSync(clientPath, { recursive: true })
       }
     }
 
-    writeFileSync(outputSchemaPath, generated.schema)
-
-    writeFileSync(`${outputResolverPath}/index.ts`, generated.resolvers.index)
-    
-    if(!existsSync(`${outputResolverPath}/custom`)) {
-      mkdirSync(`${outputResolverPath}/custom`)
+    if(!existsSync(`${pathForSchema}`)) {
+      mkdirSync(`${pathForSchema}`, { recursive: true })
     }
-
-    if(!existsSync(`${outputResolverPath}/generated`)) {
-      mkdirSync(`${outputResolverPath}/generated`)
+    if(!existsSync(`${customResolvers}/custom`)) {
```

Comment:
Why not use `./src/resolvers/custom` directly in paths instead of appending it here. Same for generated resolvers.